### PR TITLE
gather_job_data: added `--since` parameter

### LINKED ISSUE #39: Make a workflow statistics dashboard
Show the following data, grouped by 1-week intervals:

* Number of total runs and total failures
* Number of each detected failure type, all on one chart

### LINKED ISSUE #43: gather_job_data: add --since parameter
Add `--since` parameter to filter processed jobs:

```bash
./multivac/gather_job_data.py --since ###[d|h]
# process only jobs which ran in the last two days, skip the rest
./multivac/gather_job_data.py --since 2d
# last 5 hours
./multivac/gather_job_data.py --since 5h
```

Working with absolute dates is not needed (yet).

### PR DESCRIPTION
**Pull request by lastoCHka42**
This patch adds parameter to filter jobs to gather by data. If option set, script will gather data only about jobs for last N days (if set Nd argument) or last N hours (if set Nh argument).

This argument allows to avoid gathering all data if we already have such data (i.e. when put data to InfluxDB).

Resolves #43 
Resolves #39

### CODE REVIEW

**Review by NickVolynkin — APPROVED** [review]

File: multivac/gather_job_data.py [hunk 1]
```diff
@@ -83,13 +83,33 @@ def __init__(self, cli_args):
...
+
+        since: str = cli_args.since
+        if since and len(since) > 1:
+            wrong_usage_message = 'Wrong \'--since\' option: got wrong %s. ' \
+                                  'Usage: NN[d|h], example: 42d.'
+            if since.endswith('d'):
+                try:
+                    self.since_seconds = int(since.rstrip('d')) * 86400
+                except ValueError:
+                    print(wrong_usage_message % 'number')
```
Comment (NickVolynkin) [inline multivac/gather_job_data.py:96]: please use f-strings, we're already using them in this repo

File: docs/gather_job_data.rst [hunk 2]
```diff
@@ -51,6 +51,14 @@ To gather data from a number of most recent workflows, use `--latest`:
 
     $ ./multivac/gather_job_data.py --latest 1000
 
+To gather data for last N days or N hours, use ``--since``:
+
+..  code-block:: console
+
+    $ # see data for last week (7 days)
```
Comment (NickVolynkin) [inline docs/gather_job_data.rst:58]:
```suggestion
    $ # see data for the last week (7 days)
```

File: docs/gather_job_data.rst [hunk 3]
```diff
@@ -51,6 +51,14 @@ To gather data from a number of most recent workflows, use `--latest`:
...
+    $ ./multivac/gather_job_data.py --since 7d
+    $ # see data for last 12 hours
```
Comment (NickVolynkin) [inline docs/gather_job_data.rst:60]:
```suggestion
    $ # see data for the last 12 hours
```

File: README.md [hunk 4]
```diff
@@ -8,6 +8,133 @@ It collects data about workflow runs and workflow run jobs from
...
+            A workflow run list page to start from it. Default: 1
+
+
+EXAMPLE
+
+    Collect all data about workflow runs and workflow run jobs till the end
+    of rate limit from repository `tarantool/multivac` branches `master` and
+    `sample-branch`, but don't collect logs:
+
+```bash
```
Comment (NickVolynkin) [inline README.md:53]: If you start lines with $ or #, it's console, not bash

File: README.md [hunk 5]
```diff
@@ -8,6 +8,133 @@ It collects data about workflow runs and workflow run jobs from
...
+    --short
+
+            Coalesce 'runs-on' labels by a first word
+
+EXAMPLE
+
+    Generate a report in the HTML format for branches `master` and
+    `sample-branch`:
+
+```bash
```
Comment (NickVolynkin) [inline README.md:88]: Same as above

File: multivac/gather_job_data.py (hunk 1 above)
Comment (lastoCHka42) [reply multivac/gather_job_data.py]: Fixed with `.format()`

File: README.md (hunk 4 above)
Comment (lastoCHka42) [reply README.md]: Fixed

File: README.md (hunk 5 above)
Comment (lastoCHka42) [reply README.md]: Fixed

File: docs/gather_job_data.rst (hunk 3 above)
Comment (lastoCHka42) [reply docs/gather_job_data.rst]: fixed

File: docs/gather_job_data.rst (hunk 2 above)
Comment (lastoCHka42) [reply docs/gather_job_data.rst]: fixed

**Review by NickVolynkin — APPROVED** [review]

**Pull request merged**